accessibility-swarms.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: Accessibility Swarms
+---
+Accessibility Swarms are a practice that we carry out in BBC News.
+
+Swarms take place on **all new front-end code**, this means any changes to HTML, CSS or JavaScript code.
+
+We do this because **accessibility is a team responsibility**.
+
```

Comment:
It'd be good to add something like: 
```suggestion
As a result of the swarm, we expect to 
- note any accessibility issues that arise as a result of the new work, 
- prioritise these issues
- write up tickets to fix the issues, adding relevant information from the swarm.
```

Reply:
I think that is covered in the 'how to carry out a swarm section'.